### PR TITLE
Distance

### PR DESCRIPTION
**Pull request by ivanko22**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ app.use(express.static(__dirname + "/views"));
 
 let cssOptions = {
     recursive: true,
-    minimize: false
+    minimize: true
```
Comment (MorganLee909) [inline app.js:16]: Leave this false. We only want to minimize in production.

File: views/js/models/Vendor.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ class Vendor {
         this._items = [];
         this._address = (address === undefined) ? "" : address;
         this._phone = (phone === undefined) ? "" : phone;
+        this._distance = `${(distance * 0.001).toFixed(1)} mi`;
```
Comment (MorganLee909) [inline views/js/models/Vendor.js:14]: Why did you change this to kilometers? I told you to assume that it was miles and that I would fix it.

**Review by MorganLee909 — CHANGES_REQUESTED** [review]
You need to store the number on the class, not a string that you want displayed. Classes are for storing the data.

Also, the branch is named 'distance' but you made changes to the css in the components. I'll accept it this time, but in the future, I will refuse PR's that do multiple different things.

File: app.js (hunk 1 above)
Comment (ivanko22) [reply app.js:16]: done

**Comment by ivanko22** [comment]
updated

File: views/js/pages/searchResults.js [hunk 3]
```diff
@@ -73,7 +73,7 @@ let searchResultsPage = {
             item.vendor = vendors[i];
             item.setAttribute('vendor', vendors[i].name);
             item.setAttribute('address', vendors[i].address);
-            item.setAttribute('distance', vendors[i].distance);
+            item.setAttribute('distance', `${(vendors[i].distance * 0.001).toFixed(1)} mi` );
```
Comment (MorganLee909) [inline views/js/pages/searchResults.js:76]: You are still converting to km for some reason.  Remember, I said to assume that distance is being returned in miles? I will update the backend before merging to send miles. Just don't multiply the distance by anything.

**Comment by MorganLee909** [comment]
You are still converting to km for some reason. Just display the distance, don't multiply it by anything.

**Review by MorganLee909 — APPROVED** [review]

**Pull request merged**